charts/localstack/test-values.yaml

```
@@ -20,6 +20,12 @@ mountDind:
   forceTLS: true
 
 
+ingressHost: chart-example.local
+ingress:
+  enabled: true
+  hosts:
+    - host: {{ .Values.ingressHost }}
```

This actually fails. You need to put the value in quotes, otherwise the rendering fails with the following error:
```bash
$ helm template ls-helm-demo ./charts/localstack/ -f charts/localstack/test-values.yaml > result.yaml
Error: failed to parse charts/localstack/test-values.yaml: error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{".Values.ingressHost":interface {}(nil)}
```
```suggestion
    - host: "{{ .Values.ingressHost }}"
```

---

Apologize for this, I was testing my changes with another value files. 
Will make sure to also test with the `test-values.yaml` file. 
Fixed